CHANGELOG.md

```diff
@@ -36,15 +36,15 @@
 ### E2E Encryption Router Termination
```

Comment:
@andrewpmartinez I updated the `xgress_edge_transport` references to `edge_transport`